superset-frontend/tools/eslint-plugin-theme-colors/package.json

```diff
@@ -9,9 +9,5 @@
   "keywords": [],
   "license": "Apache-2.0",
   "author": "Apache",
-  "dependencies": {},
-  "engines": {
-    "node": "^16.9.1",
-    "npm": "^7.5.4"
```

Comment:
This was just bumped to support npm 8 in [this](https://github.com/apache/superset/pull/19365) PR. Not sure if it's _necessary_ but thought I should ping @villebro in case it is. 

Happy to approve this otherwise, though if this doesn't become an objection. Thanks for tending to this!